### DISCUSSION TITLE
Cross-fetch error with Vite & Nuxt3

### DISCUSSION THREAD

**Discussion by eMeRiKa13** [supabase/supabase#4805] [Questions]
Hi,

I'm using Supabase and the js library with Nuxt3 which use [Vite ](https://vitejs.dev/)behind the scene.

Unfortunately since a newer version of Nuxt3 and Vite, my project can't work anymore due to the Supabase dependency cross-fetch.

I have the following error `require is not defined` in the [cross-fetch code](https://github.com/lquixada/cross-fetch/search?q=const+nodeFetch+%3D+require%28%27node-fetch%27%29) because Vite deprecated 'require' and now ask for 'import' (if I understood well...).

Unfortunately, [an issue about that problem](https://github.com/lquixada/cross-fetch/issues/108) is open since May 2021 in the cross-fetch Github without answer.

So here I am. How I can fix this problem and continue to work?

Thanks a lot

**Comment by abeldebruijn** [comment]


**Reply by eMeRiKa13** [reply]
Thank you. What can I use as email and password to try?

Where do you use Supabase? I'm trying to understand where the data came from. I see in your Blog model:

```js
const { data } = await nuxtApp.$httpClient(`v1/news/items/${id}`);`
```

What is `v1/news/items/`?
